frontend/src/views/mspcontracts/UpdateContractAddress.vue

```diff
@@ -134,16 +162,18 @@ export default {
       homeAddress: {
         addressLine1: '',
         addressLine2: '',
-        addressLine3: '',
-        addressLine4: '',
         postalCode: '',
+        country: 'Canada',
```

Comment:
It's a little weird that the homeAddress country is 'Canada' and the mailingAddress country is 'CA'. But it's fine because the data isn't set anyway and this is just an artifact of the fact that one is a text input and the other a dropdown where the values are bound to the code.